themes/custom/droopler_theme/scss/components/_d_p_carousel.scss

```diff
@@ -8,7 +8,7 @@ $d-p_carousel-item-text-bold-color: $color-troilus !default;
 $d-p-carousel-item-line-color: $color-agamemnon !default;
 $d-p-carousel-line-color: $color-agamemnon !default;
 $d-p-carousel-background: $color-agamemnon !default;
-$d-p-carousel-arrow-color: $color-agamemnon !default;
+$d-p-carousel-arrow-color: $color-black !default;
```

Comment:
Should be `$color-cassandra` :-)

Explanation: https://github.com/droptica/droopler_project/blob/master/web/themes/custom/droopler_subtheme/scss/config/_base_theme_overrides.scss